docs/source/topics/assembly_troubleshooting.adoc

```diff
@@ -3,10 +3,9 @@
 
 [NOTE]
 ====
-The goal of {rh-prod} is to deliver an OpenShift environment for development and testing purposes.
-Issues occurring during installation or usage of specific OpenShift applications are outside of the scope of {prod}.
+The goal of {rh-prod} is to deliver an {ocp} environment for development and testing purposes.
```

Comment:
`{rh-prod}` currently expands to `Red Hat CRC` which is not what we want.

Reply:
Upstream, correct. Downstream, we have both `{prod}` and `{rh-prod}` defined as the exact same content: "Red Hat OpenShift Local." There's no duplication of the term across versions.

Are you suggesting that we shouldn't mention "Red Hat CRC" at all? If so, we can redefine the contents of the attribute in `meta/product-attributes.adoc`.

Reply:
upstream is just `CRC`

Reply:
Updated this PR to remove the "Red Hat" from "Red Hat CRC".